Tests iOS/MyList/SavesTests.swift

```diff
@@ -226,12 +226,15 @@ class SavesTests: XCTestCase {
         ]
 
         for expectedString in expectedContent {
-            let cell = app
-                .readerView
-                .cell(containing: expectedString)
-                .wait()
-
-            app.readerView.scrollCellToTop(cell)
+            if app.readerView.cell(containing: expectedString).isHittable {
+                print("The string was found:")
+                print(expectedString)
+            } else {
+                app.readerView.element.swipeUp()
+                let cell = app.readerView.cell(containing: expectedString)
+                app.readerView.scrollCellToTop(cell)
+                XCTAssertTrue(cell.exists)
+            }
```

Comment:
does this check if its hittable first before scrolling? Do we want to an some assertions here instead of prints?

Reply:
After discussion, we decided to go with a guard conditional, I'll implement that now.